### DISCUSSION TITLE
Search vertical default query string parameter in v4.4.1

### DISCUSSION THREAD

**Discussion by sw00sh0r** [microsoft-search/pnp-modern-search#1504] [Q&A] [answered]
Hi,

I am missing the option to add a default vertical querystring parameter to the Search Vertical webpart in v4.4.1, so I can pre-select one of the vertical tabs via URL-Paramater, e.g. www.contoso.sharepoint.com/sites/Demo.aspx?v=$NameOfTheTab.
In v3.15.3 it is still there:

![MicrosoftTeams-image](https://user-images.githubusercontent.com/68747426/141072819-330baf4a-d725-4c62-ab92-d7cba654d80b.png)

Has the feature been removed?

**Comment by wobba** [comment, marked as answer]
It's not implemented. Tracked in #732
